kalite/distributed/static/js/distributed/video/hbtemplates/video-player.handlebars

```diff
@@ -1,9 +1,13 @@
 <div id="video-player"{{#unless content_urls }} class="client-online-only"{{/unless}}>
     <video class="video-player video-js vjs-default-skin vjs-fullscreen" id="{{ random_id }}">
         <source src="{{ content_urls.stream }}" type="{{ content_urls.stream_type }}" />
-        {{#each subtitle_urls }}
-        <track kind="captions" src="{{ this.url }}" srclang="{{ code }}" label="{{ name }}" {{#ifcond code "==" ../selected_language }}{{#ifcond ../code "!=" "en" }}default="True"{{/ifcond}}{{/ifcond}}/>
-        {{/each}}
+        {{#current_language_is "en"}}
+            {{#ifcond translated_youtube_lang "==" "en"}}
```

Comment:
> Because I encountered some dubbed videos having subtitles.

Okay I think you're right. So I guess the point then must be that we should only use `{{#ifcond translated_youtube_lang "==" "en"}}` condition and remove `{{#current_language_is "en"}}`.

The intention would be that a video with English audio can still display with translated subtitles.